store/store.go

```diff
@@ -1955,6 +1955,7 @@ type CurrentSnap struct {
 	RefreshedDate    time.Time
 	IgnoreValidation bool
 	Block            []snap.Revision
+	RequestSeed      string
```

Comment:
given how this is used in practice, it could make more sense to pass it in once with RefreshOptions instead of per snap